apps/ledger-live-desktop/src/renderer/hooks/useActionCards.tsx

```diff
@@ -53,6 +60,13 @@ const useActionCards = () => {
     const currentCard = findCard(cardId);
     const actionCard = findActionCard(cardId);
 
+    if (actionCard?.id?.includes("testinDev")) {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-expect-error
+      actionCard.url = actionCard.id;
```

Comment:
do we need this ?

Reply:
I think so it's how it's done below

Reply:
It's needed for the `logContentCardClick`, not for the openUrl